R/QueueDataWarehouse.R

```diff
@@ -71,7 +72,7 @@
 QueueDataWarehouse <- function(reportsuite.id, date.from, date.to, metrics, elements,
                                date.granularity='day', segment.id='', data.current=TRUE,
                                expedite=FALSE, interval.seconds=5, max.attempts=120,
-                               validate=TRUE,enqueueOnly=TRUE, ftp=ftp) {
+                               validate=TRUE,enqueueOnly=TRUE, ftp=ftp, format='csv') {
```

Comment:
Thanks for the feedback Randy. Yes, I agree with you about not exposing the format argument in the public functions where possible, as it should be treated as an internal mechanism.
I don't think pagination is possible with the CSV format. In future, to support pagination, we could revisit the option of using the default JSON format once we have worked out how to parse it given the issue with the API's JSON object structure I mentioned above.